### PR TITLE
adding a route for unknown newsletters

### PR DESCRIPTION
**Pull request by melanierogan**
Now we have `/pages/newsletters/unknown-newsletter/subscribe/error` 🐿 v2.12.0

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -87,6 +87,11 @@ app.get('/pages/newsletters/:id([a-f0-9]{24})/subscribe/upgrade-premium', (req,
   res.render('oneclick/upgrade-premium', { id, name });
 });
 
+app.get('/pages/newsletters/unknown-newsletter/subscribe/error', (req, res) => {
+  const { id, name } = req.query;
```
Comment (keirog) [inline server.js]: Guess we don’t need this bit as we don’t know what newsletter it is ¯\_(ツ)_/¯

File: test/server.test.js [hunk 2]
```diff
@@ -90,6 +90,20 @@ describe('GET /upgrade-premium', () => {
   });
 }); 
 
+
+describe('GET /error', () => {
+  it.only('returns 200 status', function() {
```
Comment (keirog) [inline test/server.test.js]: Careful of these `.only`s, they’re making the rest of the tests skip
(There’s an eslint plugin to catch this; we should add that to our apps!)

Comment (melanierogan) [reply test/server.test.js]: oops, thanks! They're gone now

File: server.js (hunk 1 above)
Comment (melanierogan) [reply server.js]: haha, good point.

**Pull request merged**